dot/core/service_integration_test.go

```diff
@@ -661,8 +579,7 @@ func TestService_HandleCodeSubstitutes(t *testing.T) {
 
 	s.blockState.StoreRuntime(blockHash, rt)
 
-	ts := rtstorage.NewTrieState(nil)
-
```

Comment:
I am going to guess that removed lines have happened because of leftovers after debugging and not very deliberate. revert them since they provide readability.

Reply:
I didn't ask to change anything here. I just wanted you to add the empty line back.